docs/user-guide/binned-data/histogramming-grouping-and-binning.ipynb

```diff
@@ -71,14 +71,15 @@
     "\n",
     "## Histogramming\n",
     "\n",
-    "`histogram` directly sums the data and is efficient.\n",
+    "[scipp.hist](../../generated/functions/scipp.hist.rst) directly sums the data and is efficient.\n",
     "Limitations are:\n",
     "\n",
-    "- Can currently only histogram along a single dimension.\n",
-    "- Can currently only apply the \"sum\" to accumulate into a bin.\n",
-    "  Support for more operations, such as \"mean\", is considered for the future.\n",
+    "- When histogramming in more than one dimension, the implementation uses `sc.bin` internally, which may be less efficient and uses more memory.\n",
```

Comment:
Less than what? Is there an alternative?

Reply:
Less than `hist`. No alternative.